app/src/main/java/org/kiwix/kiwixmobile/intro/CustomViewPager.kt

```diff
@@ -0,0 +1,80 @@
+/*
+ * Kiwix Android
+ * Copyright (c) 2019 Kiwix <android.kiwix.org>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+package org.kiwix.kiwixmobile.intro
+
+import android.content.Context
+import android.util.AttributeSet
+import android.util.Log
+import android.view.animation.Interpolator
+import android.widget.Scroller
+import androidx.viewpager.widget.ViewPager
+
+/**
+ * A custom implementation of [ViewPager] to decrease the speed of auto-scroll animation
+ * of [ViewPager].
+ */
+const val DURATION_MULTIPLIER = 3
+
+class CustomViewPager : ViewPager {
```

Comment:
I would recommend looking up the `@JvmOverloads` annotation and using it here.

Then you can call `postInitViewPager` in an `init` block